1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,44 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+//variables
+const quoteP = document.querySelector("#quote"),
+  authorP = document.querySelector("#author"),
+  newQuoteBtn = document.querySelector("#new-quote"),
+  autoGenerate = document.querySelector(".checkbox");
+let intervalID;
+
+// eventListeners
+eventlisteners();
+function eventlisteners() {
```

Comment:
for eventlistener, it doesn't have to be a function. 

just 
const newQuoteBtn = document.querySelector("#new-quote");
newQuoteBtn.addEventListener("click", showQuoteOnClick);

is enough 🙂

also line 502 is unnecessary

Reply:
Thank you for your tips. I learnt it from a course and I thought It could be a good way for having  more readable code.